src/classes/class-se-block-variations.php

```
@@ -165,6 +227,56 @@ private function set_event_query_args( $args, $feed_type, $feed_order = 'ASC' )
+		}
+
+		// Skip if treating each date as own event
+		if ( se_event_treat_each_date_as_own_event() ) {
+			return $where;
+		}
+
+		global $wpdb;
+
+		$feed_order      = $query->query_vars['feed_order'];
+		$meta_key        = 'desc' === $feed_order ? 'se_event_date_end' : 'se_event_date_start';
+		$order_direction = 'desc' === $feed_order ? 'DESC' : 'ASC';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$order_direction'. (Unused Code Rules)

(UnusedLocalVariable)